src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -100,4 +100,11 @@ function Base.show(io::IO, ::MIME"text/plain", limiter::SubcellLimiterIDP)
         summary_box(io, "SubcellLimiterIDP", setup)
     end
 end
+
+function get_node_variables!(node_variables, limiter::SubcellLimiterIDP,
+                             ::VolumeIntegralSubcellLimiting, equations)
+    node_variables[:alpha_limiter] = limiter.cache.subcell_limiter_coefficients.alpha
+    # Note: The coefficients are uninitialized in the first solution file.
```

Comment:
Would it make sense to initialize them with 0 or so? Uninitialized values in ParaView are kinda annoying.